tests/test_motiondetector.py

```diff
@@ -74,23 +74,73 @@ async def test_properties(self):
         assert props.detections_supported == True
         assert props.object_point_clouds_supported == False
 
-    
+
     @pytest.mark.asyncio
     async def test_captureall(self):
         md = self.getMD()
-        out = await md.capture_all_from_camera("test",return_image=True, 
+        out = await md.capture_all_from_camera("test",return_image=True,
                                                 return_classifications=True,
                                                 return_detections=True,
                                                 return_object_point_clouds=True)
         assert isinstance(out, CaptureAllResult)
         print(out)
-        assert out.image is not None 
-        assert out.classifications is not None 
+        assert out.image is not None
+        assert out.classifications is not None
         assert len(out.classifications) == 1
         assert out.classifications[0]["class_name"] == "motion"
-        assert out.detections is not None 
+        assert out.detections is not None
         assert out.detections[0]["class_name"] == "motion"
-        assert out.objects is None 
+        assert out.objects is None
 
 
+    @pytest.mark.asyncio
+    async def test_captureall_not_too_large(self):
+        md = self.getMD()
+        md.max_box_size = 1000000000
+        out = await md.capture_all_from_camera("test",return_image=True,
+                                                return_classifications=True,
+                                                return_detections=True,
+                                                return_object_point_clouds=True)
+        assert isinstance(out, CaptureAllResult)
+        print(out)
+        assert out.image is not None
+        assert out.classifications is not None
+        assert len(out.classifications) == 1
+        assert out.classifications[0]["class_name"] == "motion"
+        assert out.detections is not None
+        assert out.detections[0]["class_name"] == "motion"
+        assert out.objects is None
```

Comment:
I'm conflicted about copying and pasting test code. On the one hand, it's hard to see what is different between one test and another, and future changes need to be done multiple times in multiple tests. On the other hand, if a test fails, I want it to be very obvious which test failed for what reason, rather than just "some helper function used in many places had trouble, and it's up to you to figure out what led up to that."

I'll refactor out the shared code if you want.

Reply:
...no, I should just refactor out the shared code. When using `make test`, I get excellent stack traces describing where we came from, which should be plenty to figure out a failing test.